backend.py

```diff
@@ -122,7 +136,7 @@ def generate_single_current_value(pnp_is: float, pnp_n: float, npn_is: float, np
     outtext = read_file_as_string(outfile)
     out_data = parse_output_data(outtext)
     for voltage, current in out_data:
-        if voltage == 5.0:
+        if voltage == float(input_voltage):
```

Comment:
Using shared variable "input_voltage". Please don't do that. Instead, make the function take "input_voltage" as an input.